garrysmod/gamemodes/base/entities/entities/lua_run.lua

```diff
@@ -38,7 +38,7 @@ function ENT:RunCode( activator, caller, code )
 
 	self:SetupGlobals( activator, caller )
 
-		RunString( code )
+		RunString( code, game.GetMap() )

```

Comment:
Might be better to also include the hammer ID of the lua_run entity so as to further narrow down any errors.

```
RunString( code, game.GetMap()  .. ":" .. self:MapCreationID() )
```